app/views/pages/about.html.erb

```diff
@@ -14,50 +14,50 @@
                       </div>
                     </div>
 
-                    <p>
+                    <p class="info__body">
                     <a class="usa-link usa-link--external" href="https://18f.gsa.gov/">18F</a>, in collaboration with the <a class="usa-link usa-link--external" href="https://www.fpc.gov/">Federal Privacy Council</a> and with funding from <a  class="usa-link usa-link--external" href="https://10x.gsa.gov/">10x</a>, created this dashboard to provide a faster way to find and analyze Privacy Act notices in the <a class="usa-link usa-link--external" href="https://www.federalregister.gov/">Federal Register</a>.
                     This dashboard enables targeted search of System of Records Notices (SORN), the documents that inform the public about the existence and current state of government IT systems that collect and store personally-identifiable information (PII).
                     </p>
 
-                    <p>
+                    <p class="info__body">
```

Comment:
so rather than putting a class on every body, i might put a class on the wrapper and then have the css styling on `.info__container p`

Reply:
That makes a lot of sense, thank you! I am also happy to change the class name to something more descriptive, but could not come up with it.